components/aria2c/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Pulling ubuntu bionic as base image
```

Comment:
Remove this comment since you are pulling alpine anyway.

Reply:
👍

Reply:
@JaDogg fixed 😁